config.mak.uname

```diff
@@ -623,6 +623,9 @@ ifeq ($(uname_S),NONSTOP_KERNEL)
 	SHELL_PATH = /usr/coreutils/bin/bash
 endif
 ifeq ($(uname_S),MINGW)
+	ifneq (,$(wildcard ../THIS_IS_MSYSGIT))
```

Comment:
This still uses the msysGit-specific condition, not the one that's about MSys in general (which includes, but is not limited to, msysGit).

Let's use the generic condition.